### PR TITLE
Combobox Examples with listbox popups: scroll item referenced by aria-activedescendant into view

### LINKED ISSUE #2052: When using aria-activedescendant, the focused element is not scrolled into the visible area
[Managing Focus in Composites Using aria-activedescendant](https://w3c.github.io/aria-practices/#kbd_focus_activedescendant) say:

> Move the visual focus indicator and, if necessary, scrolled the active element into view

Unfortunately, this is not even considered in the APG examples like https://w3c.github.io/aria-practices/examples/radio/radio-activedescendant.html, https://w3c.github.io/aria-practices/examples/menu-button/menu-button-actions-active-descendant.html and https://w3c.github.io/aria-practices/examples/combobox/combobox-select-only.html).

When do I notice this: e.g. when I zoom out to 320x265 px according to WCAG 1.4.10 and then navigate through the entries with the arrow keys. In some APG examples, the problem would occur even more often and even without zoom if there were more elements (e.g. radio buttons).

Related: https://github.com/w3c/aria-practices/issues/1135, https://github.com/w3c/wcag/issues/2049#issuecomment-933437898

### PR DESCRIPTION
**Pull request by jongund**
Resolves #2052 

[Preview Link: `aria-autocomplete=both`](https://raw.githack.com/w3c/aria-practices/combobox-both-scroll-to/examples/combobox/combobox-autocomplete-both.html)
[Preview Link: `aria-autocomplete=list`](https://raw.githack.com/w3c/aria-practices/combobox-both-scroll-to/examples/combobox/combobox-autocomplete-list.html)
[Preview Link: `aria-autocomplete=none`](https://raw.githack.com/w3c/aria-practices/combobox-both-scroll-to/examples/combobox/combobox-autocomplete-none.html)

### Editor's review checklist

The scope of changes for this pull request necessatates only the following reviews.

* [x] [Editorial review](https://github.com/w3c/aria-practices/wiki/Pull-Request-Review-Process#editorial_review) by @mcking65
* [x] [Accessibility review](https://github.com/w3c/aria-practices/wiki/Pull-Request-Review-Process#accessibility_review) by @a11ydoer
* [x] [Code review](https://github.com/w3c/aria-practices/wiki/Pull-Request-Review-Process#code_review) by @howard-e

### CODE REVIEW

**Comment by jongund** [comment]
@mcking65 
I have updated the combobox example to include a scroll into view feature.

**Comment by JAWS-test** [comment]
Thank you for the quick change. It works correctly now

File: examples/combobox/combobox-autocomplete-both.html [hunk 1]
```diff
@@ -128,6 +125,43 @@ <h2 id="ex_label">Example</h2>
...
+      <h2>Accessibility Features</h2>
+      <ul>
+        <li>
+          The <code>listbox</code> option referenced by <code>aria-activedescendant</code> is scrolled into view when it is not visible in the graphical rendering.  This can occur under many conditions, but is most common when people with visual impairments use a browser's zoom feature to increase the size of content.
+        </li>
+        <li>
+          To enhance perceivability  when operating the combobox, visual keyboard focus and hover are styled using the CSS <code>:hover</code> pseudo-class, and <code>focus</code> and <code>blur</code> event handlers:
+          <ul>
+            <li>
+              To make it easier to perceive focus of the textbox/button a 2px focus ring is placed around the <code>input</code> and <code>button</code> elements with 2 pixels of space between the focus ring and the combobox.
```
Comment (mcking65) [inline examples/combobox/combobox-autocomplete-both.html:138]:
```suggestion
              To make it easier to perceive focus of the textbox and button, focus creates a 2 pixel focus ring around the <code>input</code> and <code>button</code> elements with 2 pixels of space between the focus ring and the combobox.
```

File: examples/combobox/combobox-autocomplete-both.html [hunk 2]
```diff
@@ -128,6 +125,43 @@ <h2 id="ex_label">Example</h2>
...
+            </li>
+            <li>
+              To make it easier to perceive the selected listbox option a 2px border is placed above and below option.
```
Comment (mcking65) [inline examples/combobox/combobox-autocomplete-both.html:141]:
```suggestion
              To make it easier to perceive the selected listbox option, selection creates a 2 pixel border above and below the option.
```

**Review by mcking65 — CHANGES_REQUESTED** [review]

File: examples/combobox/combobox-autocomplete-both.html (hunk 1 above)
Comment (jongund) [reply examples/combobox/combobox-autocomplete-both.html]: updated per your suggestion

**Comment by jongund** [comment]
@mcking65 
I did some updating of keyboard focus styling to support high contrast modes, part of the changes required using the `onblur` event to remove a `focus` class from the container element to remove the border around the textbox and button.

**Comment by mcking65** [comment]
Accessibility review was completed during last telecon. Merging.

**Pull request merged**